src/ophys_etl/transforms/create_slapp_inputs.py

```diff
@@ -0,0 +1,270 @@
+import argschema
+import json
+import numpy as np
+from typing import List, Any
+from pathlib import Path
+import marshmallow as mm
+from marshmallow import ValidationError
+
+from ophys_etl.schemas.fields import H5InputFile
+
+
+class ExperimentSchema(argschema.schemas.DefaultSchema):
+    experiment_id = argschema.fields.Int(
+        required=True,
+        description="experiment id")
+    local_ids = argschema.fields.List(
+        argschema.fields.Int,
+        required=False,
+        cli_as_single_argument=True,
+        description="which ROIs to select from this experiment")
```

Comment:
I think I understand from the name what this refers to, but I'd add more info here about what IDs these are. Are they in the binarized_rois output/trace, etc.

Reply:
done